.github/workflows/flutter-deploy.yml

```diff
@@ -0,0 +1,95 @@
+name: Flutter Manual Deploy Workflow
+env:
+  ENVIRONMENT_FILE: .env
+
+on:
+  workflow_call:
+    inputs:
```

Comment:
can you share link of the workflow which is calling this workflow? I think you tested it on one of the projects.